src/content/Angular-CSharp/.template.config/localize/templatestrings.cs.json

```diff
@@ -17,7 +17,7 @@
   "symbols/NoHttps/description": "Určuje, jestli se má protokol HTTPS vypnout. Tato možnost platí jenom v případě, že se pro --auth nepoužívají Individual, IndividualB2C, SingleOrg ani MultiOrg.",
   "symbols/UseLocalDB/description": "Určuje, jestli se má použít LocalDB namísto SQLite. Tato možnost platí jenom v případě, že je zadáno --auth Individual nebo --auth IndividualB2C.",
   "symbols/Framework/description": "Cílová architektura pro projekt",
-  "symbols/Framework/choices/net7.0/description": "Cílový net7.0",
+  "symbols/Framework/choices/net8.0/description": "Target net8.0",
```

Comment:
When does `Target` get converted to the actual language?

Reply:
It'll be done in #108 though I'll have to deal w/ conflicts there or close that PR and let `OneLocBuild` recreate it.

@jonfortescue why does Arcade have a process that copies new / updated English strings around in builds❔ Seems redundant since `OneLocBuild` will do about the same thing

Reply:
/ping @jonfortescue on
> @jonfortescue why does Arcade have a process that copies new / updated English strings around in builds❔ Seems redundant since OneLocBuild will do about the same thing

Reply:
This is my concern as well, don't want to approve without confirmation this is doing the right thing

Reply:
@dougbu sorry for missing this earlier!

The localized xlf files are not ever actually sent off to the translators, I don't think, just the English xlf file which is treated as the "culture-neutral" xlf on which translations are based. So I don't think it's so much copying strings around as you're seeing the results of those strings propagating to translators. In short, OneLocBuild is doing this, not the arcade infra afaik.